exla/lib/exla/defn.ex

```diff
@@ -1064,6 +1064,54 @@ defmodule EXLA.Defn do
     )
   end
 
+  defp to_operator(
+         :indexed_put,
+         [target, indices, updates],
+         %{type: type},
+         state
+       ) do
+    target = to_type(target, type)
+    updates = to_type(updates, type)
+
+    args = [%{type: type, shape: {}}, %{type: type, shape: {}}]
+    scatter_fn = op_computation(:add, args, state)
```

Comment:
I'm having some trouble to find an update computation that will return the rhs, so that we can scatter in place instead of adding.

Reply:
You can build the computation from scratch, something like this:

```elixir
defp scatter_reduction_computation(state) do
  subbuilder = subbuilder(state.builder, :scatter_reduction)

  param_shape = EXLA.Shape.make_shape(type, {})
  left = EXLA.Op.parameter(subbuilder, 0, param_shape, "left")
  right = EXLA.Op.parameter(subbuilder, 1, param_shape, "right")
  
  tuple = EXLA.Op.tuple(subbuilder, [left, right])
  result = EXLA.Op.get_tuple_element(subbuilder, 1)
  
  EXLA.Builder.build(result)
end
```

Reply:
Thanks!